packages/sw-lib/src/lib/sw-lib.js

```diff
@@ -55,8 +55,37 @@ class SWLib {
   constructor({cacheId, clientsClaim, handleFetch,
                directoryIndex = 'index.html',
                precacheChannelName = 'precache-updates',
-               ignoreUrlParametersMatching = [/^utm_/],
-             } = {}) {
+               ignoreUrlParametersMatching = [/^utm_/]} = {}) {
+    if (!environment.isServiceWorkerGlobalScope()) {
+      // If we are not running in a service worker, fail early.
+      throw ErrorFactory.createError('not-in-sw');
+    }
+
+    if (environment.isDevBuild()) {
+      if (environment.isLocalhost()) {
+        // If this is a dev bundle on localhost, print a welcome message.
+        logHelper.debug({
+          message: 'Welcome to Workbox!',
+          data: {
+            '📖': 'https://googlechrome.github.io/sw-helpers/',
+            '❓': 'https://stackoverflow.com/questions/ask?tags=workbox',
```

Comment:
Same as above:

"If you have any questions, please raise them on stackoverflow with the workbox tag: <URL>"